pkg/crc/machine/bundle/metadata.go

```diff
@@ -111,13 +111,27 @@ func (bundle *CrcBundleInfo) GetInitramfsPath() string {
 }
 
 func (bundle *CrcBundleInfo) GetKubeadminPassword() (string, error) {
```

Comment:
You can delete this function in fact now. kubeadmin password in bundle will disppear. It removes the if in ~/.crc/machines/... below.

Reply:
Ok :+1: